cmd/podman/containers/ps.go

```diff
@@ -309,7 +305,7 @@ func createPsOut() ([]map[string]string, string) {
 	if listOpts.Namespace {
 		row = "{{.ID}}\t{{.Names}}\t{{.Pid}}\t{{.Namespaces.Cgroup}}\t{{.Namespaces.IPC}}\t{{.Namespaces.MNT}}\t{{.Namespaces.NET}}\t{{.Namespaces.PIDNS}}\t{{.Namespaces.User}}\t{{.Namespaces.UTS}}"
 	} else {
-		row = "{{.ID}}\t{{.Image}}\t{{.Command}}\t{{.CreatedHuman}}\t{{.Status}}\t{{.Ports}}\t{{.Names}}"
+		row = "{{.ID}}\t{{.Image}}\t{{.Command}}\t{{.CreatedHuman}}\t{{.Status}}\t{{.RestartCount}}\t{{.Ports}}\t{{.Names}}"
```

Comment:
IMO we should not change the default output, I see no reason why I would want to see  the restart count by default. Users can always supply their own format template. Also tihs breaks backwards compat and docker compat if users really on the current ordering in scripts, not a good idea regardless but I don't think this justifies breaking them.

Reply:
Added a --restart flag to display the restart count when set to true so we don't break backwards compatibility and users can use the flag or template to view the restart count for containers